todo_app/templates/index.html

```diff
@@ -8,19 +8,39 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %}
-          <li>{{item.title}}</li>
+    {% for list in lists %}
+    <div class="container">
+      <h2>{{list.name}}</h2>
+      <ul class="row">
+        {% for item in list.items %}
+        <div class="col-md-auto">
+          <div class="row-sm">
+            {{item.name}}
+          </div>
+            <form action="/update" method="put">
+              <div class="row-sm">
+              <label for="list">Status </label>
+                <select name="list_id">
+                  {% for list_option in lists %}
+                    <option {{ "selected" if list_option == list }} value="{{list_option.id}}">{{list_option.name}}</option>
+                  {% endfor %}
+                </select>
+              </div>
+              <input type="hidden" name="item_id" value="{{ item.id }}"/>
+                <button class="row-sm" type="submit" value="Submit">
+                  Update item
+                </button>
+            </form>
+        </div>
         {% endfor %}
       </ul>
     </div>
+    {% endfor %}
     <div class="col-md-12">
-      <h2>Add a new to do item</h2>
+      <h2>Add a new To Do item</h2>
       <form action="/new" method="put">
-        <label for="title">New to do: </label>
-        <input type="text" name="title" id="title" required />
+        <label for="name">New to do: </label>
```

Comment:
Minor and possibly a very conscious decision, but worth noting that here you've let Trello's choice of naming spill through and change lots of other parts of our code - if we'd just kept it as "title" in our `Item` class something like below, then we wouldn't need to change this HTML or the `app.py` quite as much. That said, happy to listen to arguments that `name` is an improvement!

```python
class Item:
   def __init__(self, id, title, list):
      self.id = id
      self.title = title
      self.list = list

# created from a Trello response still with:
item = Item(json_response['id'], json_response['name'], json_response['idList'])
```